### PR TITLE
Fix package type

### PR DESCRIPTION
**Pull request by kylekatarnls**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
     "name": "pdepend/pdepend",
     "description": "Official version of pdepend to be handled with Composer",
     "license": "BSD-3-Clause",
+    "type": "library",
```
Comment (jakzal) [inline composer.json]: Is it a library or rather a tool? 🤔

Comment (kylekatarnls) [reply composer.json]: It's not up to decide:
https://getcomposer.org/doc/04-schema.md#type

It's a library according to the restricted composer provide, this is not a free text, this is a setting to get the right behavior in composer.

Comment (ravage84) [reply composer.json]: library is fine, see the [composer.json schema docs](https://getcomposer.org/doc/04-schema.md#type) on that.

Comment (jakzal) [reply composer.json]: I see. It defaults to "library" though :P

**Pull request merged**